### PR TITLE
[agent] parse xpanid in byte array format

### PR DESCRIPTION
**Pull request by gjc13**

### CODE REVIEW

File: src/agent/ncp_wpantund.cpp [hunk 1]
```diff
@@ -108,6 +108,7 @@ DBusHandlerResult ControllerWpantund::HandlePropertyChangedSignal(DBusMessage &a
 
 otbrError ControllerWpantund::ParseEvent(const char *aKey, DBusMessageIter *aIter)
 {
+    printf("controller parse\n");
```
Comment (bukepo) [inline src/agent/ncp_wpantund.cpp]: use `otbrLog()` or remove?

File: src/agent/ncp_wpantund.cpp [hunk 2]
```diff
@@ -191,10 +192,11 @@ otbrError ControllerWpantund::ParseEvent(const char *aKey, DBusMessageIter *aIte
             dbus_message_iter_recurse(aIter, &subIter);
             dbus_message_iter_get_fixed_array(&subIter, &xpanid, &count);
             VerifyOrExit(count == sizeof(xpanid), ret = OTBR_ERROR_DBUS);
+            printf("Iter get\n");
```
Comment (bukepo) [inline src/agent/ncp_wpantund.cpp]: ditto.

File: tests/meshcop/meshcop [hunk 3]
```diff
@@ -458,6 +436,28 @@ ba_start()
...
+}
+
+network_form()
+{
+  readonly OT_PANID="$(random_panid)"
+  readonly OT_XPANID="$(random_xpanid)"
+  readonly OT_MASTER_KEY="$(random_masterkey)"
+  readonly OT_CHANNEL="$(random_channel)"
+
+  curl --header "Content-Type: application/json" --request POST --data "{\"networkKey\":\"${OT_MASTER_KEY}\",\"prefix\":\"fd11:22::\",\"defaultRoute\":true,\"extPanId\":\"${OT_XPANID}\",\"panId\":\"${OT_PANID}\",\"passphrase\":\"${OT_AGENT_PASSPHRASE}\",\"channel\":${OT_CHANNEL},\"networkName\":\"${OT_NETWORK_NAME}\"}" http://127.0.0.1/form_network || die "WEB: foram failed"
```
Comment (bukepo) [inline tests/meshcop/meshcop]: use single quote to avoid those escapes `\"`?

Comment (gjc13) [reply tests/meshcop/meshcop]: In single quote, `${xxx}` will be parsed as raw string.

File: src/web/Makefile.am [hunk 4]
```diff
@@ -42,6 +42,10 @@ otbr_web_LDADD                                             = \
     libotbr-web.la                                           \
 	$(NULL)
 
+otbr_web_LDFLAGS                                           = \
+    -static                                                  \
```
Comment (bukepo) [inline src/web/Makefile.am]: Align

File: tests/scripts/meshcop [hunk 5]
```diff
@@ -458,6 +437,28 @@ ba_start()
   write_syslog "AGENT: start complete"
 }
 
+web_start()
+{
+  write_syslog "WEB: kill old"
+  sudo killall "${OTBR_WEB}" || true
+  write_syslog "WEB: starting"
+  (cd "${ORIGIN_PWD}" && sudo "${OTBR_WEB_PATH}" -I "${TUN_NAME}" -p 8773 -a 127.0.0.1&)
+  sleep 20
```
Comment (bukepo) [inline tests/scripts/meshcop]: Do we need wait such long?

File: tests/scripts/meshcop [hunk 6]
```diff
@@ -458,6 +437,28 @@ ba_start()
...
+}
+
+network_form()
+{
+  readonly OT_PANID="$(random_panid)"
+  readonly OT_XPANID="$(random_xpanid)"
+  readonly OT_MASTER_KEY="$(random_masterkey)"
+  readonly OT_CHANNEL="$(random_channel)"
+
+  curl --header "Content-Type: application/json" --request POST --data "{\"networkKey\":\"${OT_MASTER_KEY}\",\"prefix\":\"fd11:22::\",\"defaultRoute\":true,\"extPanId\":\"${OT_XPANID}\",\"panId\":\"${OT_PANID}\",\"passphrase\":\"${OT_AGENT_PASSPHRASE}\",\"channel\":${OT_CHANNEL},\"networkName\":\"${OT_NETWORK_NAME}\"}" http://127.0.0.1:8773/form_network || die "WEB: foram failed"
```
Comment (bukepo) [inline tests/scripts/meshcop]: Typo: foram

File: .travis/script.sh [hunk 7]
```diff
@@ -52,6 +52,10 @@ posix-check)
     ;;
 
 meshcop)
+    if [ $(command -v gcc-5) ]; then
```
Comment (bukepo) [inline .travis/script.sh]: Just `if command -v gcc-5; `?

File: src/web/Makefile.am (hunk 4 above)
Comment (gjc13) [reply src/web/Makefile.am]: Already aligned?

**Pull request merged**